library/src/main/scala/scala/scalajs/js/timers/package.scala

```diff
@@ -14,10 +14,11 @@ import scala.concurrent.duration.FiniteDuration
 /**
  *  <span class="badge badge-non-std" style="float: right;">Non-Standard</span>
  *  Non-standard, but in general well supported methods to schedule asynchronous
- *  exeuction.
+ *  execution.
  *
  *  The methods in this package work in all JavaScript virtual machines
- *  supported by Scala.js (currently Rhino, Node.js and PhantomJS).
+ *  supported by Scala.js (currently Node.js and PhantomJS in the core, plus
+ *  Selenium in an external repository), as well as all browsers.
```

Comment:
Selenium is not really a JavaScript virtual machine. The underlying browser is (which arguably *must* support this, because it's part of the DOM).

I would suggest we reword this to say something like:
"The methods in this package work in all JavaScript virtual machines supporting `setTimeout` and `setInterval`."